src/monetization.js

```diff
@@ -169,6 +170,40 @@ export class Monetization extends EventEmitter {
         return data;
     }
 
+    /**
+     * Checks if the user has access to a set of products or features.
+     * @param {array} pids - which products/features to check
```

Comment:
NIT: not sure if `pids` is good name, wouldn't it be better to use full name `productsIds`?